pkg/api/handlers/compat/containers.go

```diff
@@ -148,16 +149,23 @@ func ListContainers(w http.ResponseWriter, r *http.Request) {
 			containers = containers[:query.Limit]
 		}
 	}
-	var list = make([]*handlers.Container, len(containers))
+	list := make([]*handlers.Container, len(containers))
```

Comment:
Can we turn this into `var list []*handlers.Container` and use `append = ....` in the loop?  It'll be slightly less efficient but more robust.

Reply:
Now golangci-lint complains, should I add `//nolint`?

Reply:
Does it also complain when turning it into `list := []*handlers.Container{}`?  You can run it locally with `make lint`.

Reply:
That works.

Reply:
The best way is to preallocate with `make([]*handlers.Container, 0, len(containers))`. This allows you to use append correctly.

Reply:
Great idea, @Luap99 !